packages/vue-app/template/App.js

```diff
@@ -304,7 +304,7 @@ export default {
       if (base && route.startsWith(base)) {
         route = route.substr(base.length)
       }
-      return withoutTrailingSlash(parsePath(route).pathname)
+      return withLeadingSlash(withoutTrailingSlash(parsePath(route).pathname))
```

Comment:
I think somewhere else we are wrongly stripping leading slash from `route`. This fixes issue not the root cause

Reply:
That's a good point. Will track it down

Reply:
I think it's the `route.substr` call in the previous line - it subtracts the default router base which is `/` so we lose the leading slash. Not sure why this wasn't causing us issues before.

Reply:
So I think this PR is good to go.

Reply:
Yep :) We may use substr(base.lengh, -1) since router base is normalized with trailing slash in nuxt options.

... Adding `withBase` `withoutBase` utils to ufo